lib/bundler/installer.rb

```diff
@@ -1,6 +1,6 @@
 require 'erb'
 require 'rubygems/dependency_installer'
-require 'bundler/worker'
+require 'bundler/installer/parallel_installer'
 
 module Bundler
   class Installer < Environment
```

Comment:
Put the autoload at the top, here: `autoload :ParallelInstaller, 'bundler/installer/parallel_installer'`